### PR TITLE
Add Elastic CVE IDs

### PR DESCRIPTION
**Pull request by joshbressers**

### CODE REVIEW

File: 2018/17xxx/CVE-2018-17245.json [hunk 1]
```diff
@@ -1,18 +1,63 @@
...
+      {
+        "url": "https://www.elastic.co/community/security"
+      }
+    ]
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "Yuri Astrakhan and Nick Peihl of Elastic discovered Kibana versions 4.0 to 4.6, 5.0 to 5.6.12, and 6.0 to 6.4.2 contain an error in the way authorization credentials are used when generating PDF reports. If a report requests external resources plaintext credentials are included in the HTTP request that could be recovered by an external resource provider."
```
Comment (cve-team) [inline 2018/17xxx/CVE-2018-17245.json]: If you could for the IDs in this submission and in future submissions, please remove any names of individuals associated with discovering the vulnerability from the description. So in this case for example, the description could begin with "Elastic's Kibana versions..." or something similar would suffice. The rest here is great though. Thanks!

File: 2018/17xxx/CVE-2018-17247.json [hunk 2]
```diff
@@ -1,18 +1,63 @@
...
+      {
+        "url": "https://www.elastic.co/community/security"
+      }
+    ]
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "Elasticsearch Security versions 6.5.0 and 6.5.1 contain an XXE flaw in Machine Learningï¿½s find_file_structure API. If a policy allowing external network access has been added to Elasticsearchï¿½s Java Security Manager then an attacker could send a specially crafted request capable of leaking content of local files on the Elasticsearch node. This could allow a user to access information that they should not have access to."
```
Comment (cve-team) [inline 2018/17xxx/CVE-2018-17247.json]: Looks like there are a couple of character encoding issues with "Learning's" and "Elasticsearch's".

File: 2018/17xxx/CVE-2018-17246.json [hunk 3]
```diff
@@ -1,18 +1,64 @@
...
+      {
+        "url": "https://www.elastic.co/community/security"
+      }
+    ]
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "Nethanel Coppenhagen of CyberArk Labs discovered Kibana versions before 6.4.3 and 5.6.13 contain an arbitrary file inclusion flaw in the Console plugin. An attacker with access to the Kibana Console API could send a request that will attempt to execute javascript code. This could possibly lead to an attacker executing arbitrary commands with permissions of the Kibana process on the host system.\r\n\r\n"
```
Comment (cve-team) [inline 2018/17xxx/CVE-2018-17246.json]: Same thing here as with the previous CVE ID. Something like "Elastic's Kibana versions..." would work.

**Comment by joshbressers** [comment]
OK, the changes that are needed should all be there @cve-team

**Pull request merged**